libraries/render-utils/src/AmbientOcclusionEffect.h

```diff
@@ -16,64 +16,76 @@
 
 #include "render/DrawTask.h"
 
+class AmbientOcclusionEffectConfig : public render::Job::Config {
```

Comment:
render::Job::Config should be macro'ed, and the members should all be protected. The class that it configures should be a friend class.